File: web-services/services/auditing/src/main/java/com/cryptocaddy/services/auditing/model/request/RequestCreateAccount.java
package com.cryptocaddy.services.auditing.model.request;

public class RequestCreateAccount {

    private String email;
    private String token;

    public RequestCreateAccount(String email, String token) {
        this.email = email;
        this.token = token;
    }

    public RequestCreateAccount() {
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }
}

File: web-services/auditing/src/main/java/com/cryptocaddy/services/auditing/api/dto/request/RequestCreateAccount.kt
package com.cryptocaddy.services.auditing.api.dto.request

class RequestCreateAccount {

    var email: String? = null
    var token: String? = null

    constructor(email: String, token: String) {
        this.email = email
        this.token = token
    }

    constructor() {}
}
